cognite/client/utils/_auxiliary.py

```diff
@@ -56,6 +56,7 @@ def fast_dict_load(
     except TypeError:
         instance = cls()
     # Note: Do not use cast(Hashable, cls) here as this is often called in a hot loop
+    # Accepted: {camel_case(attribute_name): attribute_name}
```

Comment:
Test code?
```suggestion
```

Reply:
La til kommentar for å forstå hva som skjer fordi det forrvirret meg.